### PR TITLE
Add hint of Issue #478 to error text 

### PR DESCRIPTION
**Pull request by DuncanBetts**
- [ master ] Target: for bug or doc fixes, target `master`; for new features, target `features`;

- [ Not Done Yet ] Add yourself to `AUTHORS`
- [ Not Done Yet ] Add a new entry to `CHANGELOG.rst`

RST Formatted, as requested:
`Issue 478 <https://github.com/pytest-dev/pytest/issues/478>`

[Issue 478] (https://github.com/pytest-dev/pytest/issues/478)

I have implemented enkore's suggestion.

Feedback, suggestions, and rejection, all welcome! This is my first pull request :tada:.

Thanks @DuncanBetts for the PR (https://github.com/DuncanBetts).

### CODE REVIEW

**Comment by nicoddemus** [comment]
>  This is my first pull request 🎉

Congrats on your first PR! 🎈 

The fix looks good! Could you also add yourself to `AUTHORS` and a new `CHANGELOG` entry?

Thanks!

File: CHANGELOG.rst [hunk 1]
```diff
@@ -11,7 +11,7 @@
 * Cope gracefully with a .pyc file with no matching .py file (`#2038`_). Thanks
   `@nedbat`_.
 
-*
+* Add hint to error message hinting possible missing __init__.py (`#478`_). Thanks `@DuncanBetts`_.
```
Comment (nicoddemus) [inline CHANGELOG.rst]: You need to add the actual link to your user, see the lines below for `@nedbad_`:

```
.. _@DuncanBetts: https://github.com/DuncanBetts
```

(Add it before nedbat's to keep the list sorted)

**Comment by DuncanBetts** [comment]
I committed the AUTHORS and CHANGELOG in a separate commit, and then ran interactive rebase and fixed up/squashed the new commit into the old commit, which has changed the SHA I believe. When I tried to push I was behind HEAD, pull resulted in a 3 way merge. I think I did the right thing?

File: CHANGELOG.rst (hunk 1 above)
Comment (DuncanBetts) [inline CHANGELOG.rst]: Ok

**Comment by nicoddemus** [comment]
> When I tried to push I was behind HEAD, pull resulted in a 3 way merge. I think I did the right thing?

Yes no problem. If you want to avoid the merge, use `git fetch` and `git rebase upstream/master` first so you will rebase on the most up-to-date version of the repository (assuming you added `pytest-dev/pytest` as a remote named `upstream`). Then you should be able to `git push` cleanly.

**Comment by DuncanBetts** [comment]
I followed your advice @nicoddemus, it still triggered a fresh three way merge and there's an extra line in the CHANGELOG as a result.
Most likely because in an effort to keep this to a single commit, I used commit --amend.
Will try and fix now.

**Comment by nicoddemus** [comment]
Hmm that's strange, it shouldn't have applied cleanly. Here are the steps in case they were not clear from my previous explanation:

1. Add the official repository as `upstream`:

  ```
  $ git remote add upstream https://github.com/pytest-dev/pytest
  ```

2. Follow this steps to rebase:

  ```
  $ git fetch --all
  $ git rebase upstream/master
  ```

If you get a conflict somewhere, fix the conflict by following the instructions from git.

(Sorry if I'm stating the obvious, I don't know your level of expertise on git 😁 )

**Pull request closed without merging**